python/nano/example/pytorch/pl-cat-vs-dog/pl-cat-vs-dog.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
You should add the license header.

See other files for reference.

Reply:
Added